creator-node/sequelize/migrations/20201202232810-add-clockrecord-sequelize-hooks.js

```diff
@@ -0,0 +1,98 @@
+'use strict'
+
+/**
+ * TODO comment
+ */
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.addHook('beforeCreate', 'clockRecordBeforeCreate', async function (instance, options) {
+      if (instance.constructor.name !== 'ClockRecord') { return }
+
+      const clockRecord = instance
+      const clock = clockRecord.clock
+
+      // DBManager calls ClockRecord.create() with a subquery for clock, which breaks this pattern
```

Comment:
This only applies to the secondaries, as the primaries pass in a subquery as described here

Reply:
yep you right - tried to make comment more explicit. lmk wyt